Code/RDGeneral/types.cpp

```diff
@@ -46,9 +46,6 @@ const std::string _MolFileBondAttach = "_MolFileBondAttach";
 const std::string _MolFileBondType = "_MolFileBondType";
 const std::string _MolFileBondStereo = "_MolFileBondStereo";
 const std::string _MolFileBondCfg = "_MolFileBondCfg";
-const std::string _NonExplicit3DChirality =
```

Comment:
this needs to be removed from types.h too. I think that's why the swig builds are failing

Reply:
done